recipes-devtools/opkg/files/0002-opkg-check-temp-dir-exists-before-delete.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/libopkg/opkg_conf.c b/libopkg/opkg_conf.c
```

Comment:
Could you add the commit message to the .patch?

https://github.com/ni/nilrt/blob/nilrt/master/hardknott/docs/CONTRIBUTING.md#patch-files